src/schedule/MongoSchedule.ts

```diff
@@ -21,7 +26,9 @@ export class MongoSchedule extends Schedule {
   private constructor(
     protected readonly scheduleId: string,
     protected readonly connection: Connection,
-    pingIntervalMs: number
+    pingIntervalMs: number,
+    // @ts-ignore
```

Comment:
Why the ignore? IJ doesn't complain and `npm run build` executes fine on my machine. 🤔